build/split/data_split.py

```diff
@@ -0,0 +1,33 @@
+import os
+import shutil
+from random import shuffle
+from math import floor
+
+def get_file_list_from_dir(datadir):
+	all_files = os.listdir(os.path.relpath(datadir))
+	data_files = list(filter(lambda file: file.endswith('.png'), all_files))
+	return data_files
+
+def randomize_files(file_list):
+	shuffle(file_list)
+	return file_list
+
+def get_training_and_testing_sets(file_list):
+	split = 0.7
+	split_index = floor(len(file_list) * split)
+	training = file_list[:split_index]
+	testing = file_list[split_index:]
+	return training, testing
+
+def main():
+	datadir = "./data"
+	data_files = get_file_list_from_dir(datadir)
+	randomized = randomize_files(data_files)
+	training, testing = get_training_and_testing_sets(randomized)
+	for f in training:
+		shutil.move('./data/' + f, "./train/")
+	for f in testing:
+		shutil.move('./data/' + f, "./test/")
```

Comment:
Would recommend having a validation split as well and instead of hard-coding the split fraction, u could make it an input parameter...

Reply:
Yea, I'm actually working on a preprocessing script that will take in an input parameter to do the training, validation, and test split at the moment. It should also take care of all the image warping, image squaring, png to jpg,  etc. so that I can use it for pix2pix.